### ISSUE TITLE
5.3.1 Issue: "Visual Studio config issue: {}"

### ISSUE THREAD

**Issue by ravishivt** [TypeStrong/grunt-ts#321]
Just updated to 5.3.1 from 5.3.0-beta.3 and hit a build issue.

```
Running "ts:src" (ts) task
Visual Studio config issue: {}
Warning: Task "ts:src" failed. Use --force to continue.

Aborted due to warnings.
```

My grunt ts:src task:

```
    ts: {
      src : {
        src: ["src/**/*.ts"],
        options: {
          fast: 'always',
          inlineSources: true,
          compiler: './node_modules/typescript/bin/tsc'
        }
      }
    }
```

**Comment by riggerthegeek** [comment]
I'm also getting this.  Config below (working in v5.3.0-beta3 and -beta4)

        ts: {
            options: {
                compiler: "./node_modules/typescript/bin/tsc",
                declaration: true,
                module: "commonjs",
                moduleResolution: "node",
                noImplicitAny: true,
                preserveConstEnums: true,
                removeComments: true,
                sourceMap: true,
                target: "es5"
            },
            all: {
                options: {
                    fast: "never"
                },
                outDir: "./<%= config.tmp %>/compiled",
                src: [
                    "*.ts",
                    "./!(<%= ignorePaths %>)/**/*.ts",
                    "./<%= config.test %>/**/*.ts"
                ]
            },
            src: {
                options: {
                    rootDir: "./"
                },
                src: [
                    "*.ts",
                    "./!(<%= ignorePaths %>)/**/*.ts"
                ]
            }
        }

**Comment by nycdotnet** [comment]
Thanks for the report.  I will take a look asap.  If it was working in 5.3.0-beta.3, please revert to that version in the meantime.

Just to confirm - neither of you are using the Visual Studio integration?  (Does not appear so bases on your gruntfiles).

Thanks.

**Comment by nycdotnet** [comment]
Interesting that your targets are both named src...

**Comment by ravishivt** [comment]
Nope, no Visual Studio integration on my end.

**Comment by nycdotnet** [comment]
This should be fixed in v5.3.2.  The issue was specifically with targets named `src`.

Thank you for the report!  You're in the patch notes.  If there is still a problem, please comment or re-open.

**Comment by riggerthegeek** [comment]
Fame at last.

In answer to your question, no I'm not using Visual Studio - I use WebStorm.  v5.3.2 has fixed the issue.  Thanks

**Comment by nishantkagrawal** [comment]
I am still seeing this issue with v5.4

**Comment by nycdotnet** [comment]
Did you rerun npm install after editing your package.json? Did 5.3.2 fix it and 5.4 was a regression? Thanks.

**Comment by davidparsson** [comment]
I've seen this from v5.3.0 to v5.4.0. I created #353 describing that.
